src/pack/fs.ts

```diff
@@ -35,7 +36,7 @@ export async function packToFs ({ input, output, blockstore: userBlockstore, has
   // Move to work dir
   if (!output) {
     const basename = typeof input === 'string' ? path.parse(path.basename(input)).name : root.toString()
-    const filename = `${basename}.car`
+    const filename = basename === "/" ? "file.car" : `${basename}.car`
```

Comment:
`file.car`  will be problematic for multiple packs in parallel as they will use same filename. Can we generate a random name like we used to do in blockstore ?

Reply:
What's the state of this discussion?
I believe that it shouldn't be an issue but LMTK.